.github/workflows/test.yml

```diff
@@ -221,7 +221,7 @@ jobs:
         name: Test hls-explicit-imports-plugin test suite
         run: cabal test hls-explicit-imports-plugin --test-options="$TEST_OPTS" || LSP_TEST_LOG_COLOR=0 LSP_TEST_LOG_MESSAGES=true LSP_TEST_LOG_STDERR=true cabal test hls-explicit-imports-plugin --test-options="$TEST_OPTS"
 
-      - if: matrix.test
+      - if: matrix.test && matrix.os != 'windows-latest'
```

Comment:
why? is there an issue?

Reply:
call-hierarchy tests were very flaky, I've fixed the problem I think and removed this.